<launch>
  <arg name="porta" default="24" />

  <param name="visao/_H" value="5" type="int" />
  <param name="visao/_S" value="220" type="int" />
  <param name="visao/_V" value="220" type="int" />
  <param name="visao/_EH" value="3" type="int" />
  <param name="visao/_ES" value="40" type="int" />
  <param name="visao/_EV" value="40" type="int" />
  <param name="visao/NUM_L" value="32" type="int" />
  <param name="visao/NUM_C" value="32" type="int" />
  <param name="visao/NUM_FRAMES" value="2" type="int" />
  <param name="visao/timeout_value" value="2" type="int" />
  <param name="visao/camera_ID" value="1" type="int" />
  <param name="visao/ref_perto" value="5" type="double" />
  <param name="visao/limiar_x" value="8" type="double" />
  <param name="visao/limiar_y" value="50" type="double" />
  <param name="visao/porc_detect" value="80" type="double" />
  <param name="visao/porta_serial" value="$(arg porta)" type="int" />
<!-- 
  local machine already has a definition by default.
       This tag overrides the default definition with
       specific ROS_ROOT and ROS_PACKAGE_PATH values
  <machine name="local_alt" address="localhost" default="true" ros-root="/u/user/ros/ros/" ros-package-path="/u/user/ros/ros-pkg" />
 -->  <!-- a basic listener node -->
  <!-- <node name="discreto_ROS" pkg="Cores" type="Cores_discr" output="screen" respawn="true" respawn_delay="0" /> -->
  <node name="continuo_ROS" pkg="Cores" type="Cores_cont" output="screen" respawn="true" respawn_delay="0" />
  <node name="esteira_comando" pkg="esteira" type="bolinhas" output="screen" />
</launch>
